docs/import.md

```diff
@@ -1,50 +1,89 @@
-# Import
 
-GGUF models and select Safetensors models can be imported directly into Ollama.
+# Importing a model
 
-## Import GGUF
+You can import a model into Ollama:
 
-A binary GGUF file can be imported directly into Ollama through a Modelfile.
+  * From Safetensors weights; or
+  * From a GGUF file
 
-```dockerfile
-FROM /path/to/file.gguf
-```
+## Importing from Safetensors
+
+Importing your model directly from the Safetensors weights is the easiest way to import a model. Ollama supports importing several different architectures including:
```

Comment:
It may be clearer to start with the instructions here:

First, create a `Modelfile`

```
FROM /path/to/safetensors
```

Then run `ollama create`:

```
ollama create my-model
```

Lastly, test the model

```
ollama run my-model
```

and then have a supported model heading below:

#### Supported models

* Llama (including Llama 2, Llama 3, and Llama 3.1);
* Phi (I think we support this now too)
* Mistral (including Mistral 1, Mistral 2, and Mixtral); and
* Gemma (including Gemma 1 and Gemma 2)